alexandria/core/tests/test_permissions.py

```diff
@@ -61,25 +63,27 @@ def has_object_permission_for_document(self, request, instance):
 
     url = reverse("document-list")
 
-    data = {
-        "data": {
-            "type": "documents",
-            "attributes": {"title": {"de": "", "en": "foo", "fr": ""}},
-        }
-    }
-
     if method in ["patch", "delete"]:
         url = reverse("document-detail", args=[doc.pk])
-        data["data"]["id"] = str(doc.pk)
 
-    if method == "patch":
-        data["data"]["attributes"]["title"] = {
-            "de": "",
-            "en": "new",
-            "fr": "",
-        }
+    data = {
+        "post": document_post_data,
+        "patch": {
+            "data": {
+                "id": doc.pk,
+                "type": "documents",
+                "attributes": {"title": {"de": "", "en": "new", "fr": ""}},
+                "relationships": {
+                    "category": {"data": {"id": doc.category.pk, "type": "categories"}}
+                },
+            }
+        },
+        "delete": None,
+    }
 
-    response = getattr(client, method)(url, data=data)
+    response = getattr(client, method)(
+        url, data=data[method], format=None if method != "post" else "multipart"
```

Comment:
Ternary could be inverted for readability

```suggestion
        url, data=data[method], format="multipart" if method == "post" else None
```

Reply:
applied